canonical-websites/www.ubuntu.com
# add webinar row on maas page

## Done

Added a webinar row to the `/server/maas` page
Also added a redirect for an alternate page URL

## QA

 - Go to `/server/maas` and check against the [copy doc](https://docs.google.com/document/d/1IrlcBMrX_TDrRC8U1_QSbEOEJ55pGY7nnCd-nfZfiXM/edit#)
 - Go to `server/provisioning" and see that you are redirected to `/server/maas`
